### PR TITLE
chore(Table): add small size

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/7315877344

### CODE REVIEW

File: packages/core/src/components/Table/Table/Table.types.ts [hunk 1]
```diff
@@ -1 +1 @@
-export type RowSizes = "medium" | "large";
+export type RowSizes = "medium" | "large" | "small";
```
Comment (YossiSaadi) [inline packages/core/src/components/Table/Table/Table.types.ts:1]:
```suggestion
export type RowSizes = "small" | "medium" | "large";
```

I'm sorry for being petty 😆

**Review by YossiSaadi — APPROVED** [review]
Malic, You're fast 😮

**Comment by YossiSaadi** [comment]
> [monday.monday.com/boards/3532714909/pulses/7315887951](https://monday.monday.com/boards/3532714909/pulses/7315887951)

fix the link
https://monday.monday.com/boards/3532714909/pulses/7315877344

File: packages/core/src/components/Table/Table/Table.types.ts (hunk 1 above)
Comment (talkor) [reply packages/core/src/components/Table/Table/Table.types.ts:1]: lol no worries, good point

**Pull request merged**